karma.conf.cjs

```diff
@@ -22,17 +22,17 @@
 
 /*global module,process*/
 
-module.exports = (config) => {
+module.exports = async (config) => {
   let webpackConfig;
   let browsers;
   let singleRun;
 
   if (process.env.KARMA_DEBUG) {
-    webpackConfig = require('./.webpack/webpack.dev.js');
+    webpackConfig = (await import('./.webpack/webpack.dev.js')).default;
```

Comment:
interesting

Reply:
yeah, CJS can't require an ESM module, but it _can_ use `await import`.